_posts/2020-12-22-ols.md

```diff
@@ -0,0 +1,76 @@
+
+The Galaxy community (users, developers, admins...) is growing all over the world and reaching new scientific fields. Time zones, and the growing number of work to manage such a big community, needs the help of a community manager in Europe working in close collaboration with other community managers around the world.
+
+At the beginning of September 2020, Beatriz started a new position at the University of Freiburg to help with the community management within the European Galaxy team. Along the 15 weeks of the program, she had the opportunity to advance her community building and project management skill while enhancing the connection with the different communities inside and outside Europe.
+
+You can watch her graduation presentation there:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/wv8_0iOMpGs?start=1639" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## [Creating a single pipeline for metagenome classification](https://openlifesci.org/ols-2/projects-participants/#creating-a-single-pipeline-for-metagenome-classification), leaded by [Muhammet Celik](https://openlifesci.org/ols-2/projects-participants#MuhammetCelik) and mentored by [Mallory Freeberg](https://openlifesci.org/ols-2#malloryfreeberg)
+
+Metagenomics has been increasingly becoming very important in studies of human and animal health. It has become clear that bacteria in and on our bodies are very significant. Thus there is a big boost in science society who are willing to study metagenomics. When we want to do downstream analysis in our microbial community, the first step is to find out about our community, what the community ultimately looks like.
+
+The idea of the project was to benchmark the state of the art tools (such as Kraken2, Centrifuge and CLARK) and create a single pipeline that will produce a single pdf file with cutoff-signifance-sensitivity values for each tool. Later we can set that pipeline on web-based system that would make the researchers easier for the scientists.
+
+The project started as a command-line tool. But after a meeting with Bérénice Batut, he decided to develop his pipeline in Galaxy. ***Add more details there***
+
+You can watch his graduation presentation there:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/IGyoiFnCvis?start=333" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+# Apply for OLS-3
+
+After successfully concluding these two cohorts, a call for applications has open for the next cohort, OLS-3, which will take place from **February to May 2021**. The deadline for application is **11 January 2021**. Visit [https://openlifesci.org](https://openlifesci.org) for details.
```

Comment:
A call for applications has opened for the next cohort, OLS-3, which will take place from...